<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CtrlNumberTable</class>
 <widget class="QWidget" name="CtrlNumberTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}
QPushButton
{
	font:14px;
    color: rgb(0,0,0);
	background-color: rgb(0, 255, 0);
	border-top: 2px solid white;	
	border-left: 2px solid white;
}
QLabel
{
color:rgb(255,255,255);
}</string>
  </property>
  <widget class="QPushButton" name="PB_N_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_10">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>10</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>3</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>5</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_7">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>7</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>6</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_0">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>0</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>4</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>9</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>8</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_N_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">	font:16px;
    color: rgb(255,255,255);
	background-color: rgb(0, 0, 0);
border:2px solid white;
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
